source/mobile/mobile-appconfig.rst

```diff
@@ -42,6 +42,9 @@ The following table shows all the configuration options that can be sent from th
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
 | vendor                 | Name of the EMM vendor or company deploying the app. Used in help text when prompting for passcodes so users are aware why the app is being protected.                                                          | String | Mattermost    |                  | Yes         | Yes                      |
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
+| inAppSessionAuth       | Use the app internal browser for SSO instead of external browser.                                                                                                                                               | String | ``false``     | ``true | false`` | Yes         | Yes                      |
```

Comment:
```suggestion
| inAppSessionAuth       | Use the app's internal browser for SSO instead of an external browser.                                                                                                                                               | String | ``false``     | ``true | false`` | Yes         | Yes                      |
```

Reply:
Thanks Carrie!